workspaces/client/app/components/layout/dashboard.tsx

```diff
@@ -1,8 +1,32 @@
-import { Outlet } from "react-router";
+import { useEffect } from "react";
+import { Outlet, useNavigate } from "react-router";
 import { css } from "styled-system/css";
+import { useAuth } from "~/hooks/useAuth";
 import { Sidebar } from "./internal/sidebar";
 
 export default function Dashboard() {
+	const { isLoading, isAuthorized, isInitialized: isInitialzied } = useAuth();
```

Comment:
```suggestion
	const { isLoading, isAuthorized, isInitialized } = useAuth();
```
あのすみません、なんですかこれは　さすがにこうしませんか

Reply:
あーこれシンボルリファクタ(VSCode)が置換諦めるやつっすね...すいません